sentinelsat/scripts/cli.py

```diff
@@ -133,6 +133,7 @@ def convert(self, value, param, ctx):
     and metadata of the returned products.
     """,
 )
+@click.option("--dhusversion", is_flag=True, is_eager=True, help="displays the DHuS version used")
```

Comment:
I would prefer a more generic `info` option. This would allow extension of other DHuS information we want to surface later.

Reply:
Done.